packages/playwright-ct-core/src/tsxTransform.ts

```diff
@@ -19,7 +19,7 @@ import path from 'path';
 import { declare, traverse, types } from 'playwright/lib/transform/babelBundle';
 import { setTransformData } from 'playwright/lib/transform/transform';
 
-import type { BabelAPI, PluginObj, T } from 'playwright/src/transform/babelBundle';
+import type { BabelAPI, PluginObj, T } from 'playwright/lib/transform/babelBundle';
```

Comment:
I think a better fix is importing types from `../../playwright/src/transform/babelBundle`. Overall, importing from `lib` is for source. We do not ship types in our packages, so importing types from `lib` seems wrong.